test_unstructured_ingest/python/test-ingest-mongodb.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/env python
+import time
+
+import click
+from pymongo.mongo_client import MongoClient
+from pymongo.operations import SearchIndexModel
+
+exact_embedding = [
```

Comment:
what was the thinking around hardcoding this? will this be brittle to any changes in partitioning (and no easy way to update like with fixtures)?

Reply:
Open to ideas, tried to come up with a test specific to the vector search index functionality. The result should be deterministic given that we're using huggingface now but this does leave the possibility that the text may change. Since the embedding is based on the text in an element, I don't see many reasons this should change, unlike the metadata.

Reply:
yea, only thing I could think is that we grab the embeddings from an element in the partitioned results file and then use that as the query?

Reply:
totally agree that this is likely not going to change often if at all, but thinking around "how bad is it" the one time it does.
if you're not feeling the suggestion I just mentioned, I guess alternatively would at least leave some documentation in case it ever breaks and code snippet for how to regenerate this.

Reply:
ya that's probably the best approach for this. Updated accordingly.

Reply:
thank you!